models/response_model.go

```diff
@@ -0,0 +1,13 @@
+package models
```

Comment:
maybe we should put it also in package `response`

Reply:
Moved